RELEASE/scripts/autoscend/iotms/clan.ash

```diff
@@ -428,6 +427,22 @@ boolean zataraClanmate(string who)
 #	}
 #	set_property("_clanFortuneConsultUses", get_property("_clanFortuneConsultUses").to_int() + 1);
 
+	int attempts = 0;
+	int player = 3038166;
+	string consultOverrideName = get_property("auto_consultChoice");
+	string name = get_player_name(player);
+	if (consultOverrideName != "")
+	{
+		name = consultOverrideName;
+		player = get_player_id(consultOverrideName).to_int();
+	}
+
+	if (!is_online(name))
```

Comment:
Are there other bots which we could try if 3038166 is down? A list of a few to try would be nice to be more robust to any being down

Reply:
I don't know of any (that are accessible in BAFH, or clans which give whitelists freely), but can add if anyone else knows of any.

Reply:
I don't know any others